dependencies/graknlabs/artifacts.bzl

```diff
@@ -17,14 +17,15 @@
 # under the License.
 #
 
-load("@graknlabs_bazel_distribution//artifact:rules.bzl", "artifact_file")
+load("@graknlabs_dependencies//distribution/artifact:rules.bzl", "native_artifact_files")
 load("@graknlabs_dependencies//distribution:deployment.bzl", "deployment_private")
-def graknlabs_grakn_core_artifact():
-    artifact_file(
+
+def graknlabs_grakn_core_artifacts():
+    native_artifact_files(
         name = "graknlabs_grakn_core_artifact",
         group_name = "graknlabs_grakn_core",
-        artifact_name = "grakn-core-server-linux-{version}.tar.gz",
+        artifact_name = "grakn-core-server-{platform}-{version}.tar.gz",
```

Comment:
This macro expands to two `artifact_file` definitions that pull in `grakn-core-server-linux` and `grakn-core-server-mac`. When you run test targets, the correct artifact for your machine's OS is selected.